internal/hcsoci/layers.go

```diff
@@ -140,11 +144,11 @@ func MountContainerLayers(ctx context.Context, layerFolders []string, guestRoot
 			uvmPath, err = uvm.AddVPMEM(ctx, layerPath)
 			if err == uvmpkg.ErrNoAvailableLocation || err == uvmpkg.ErrMaxVPMEMLayerSize {
 				log.G(ctx).WithError(err).Debug("falling back to SCSI for LCOW layer addition")
-				sm, err := uvm.AddSCSILayer(ctx, layerPath)
+				uvmPath = fmt.Sprintf(lcowSCSILayerFmt, uvm.UVMLayerCounter())
+				_, err := uvm.AddSCSI(ctx, layerPath, uvmPath, true, uvmpkg.VMAccessTypeNoop)
```

Comment:
I think we should add comments here and where we add the GPU VHD that state "we use VMAccessTypeNoop because the VM group should already have been granted access to the VHD".

Reply:
Looks like the other place already has a comment.

Reply:
Not sure if GitHub is acting up, but I don't see any difference here.